backend/app/rest/log_records_routes.py

```diff
@@ -0,0 +1,73 @@
+from flask import Blueprint, current_app, jsonify, request
+from ..middlewares.auth import require_authorization_by_role
+
+from ..services.implementations.log_records_service import LogRecordsService
+from ..services.implementations.user_service import UserService
+
+import json
+
+log_records_service = LogRecordsService(current_app.logger)
+user_service = UserService(current_app.logger)
+
+blueprint = Blueprint("log_records", __name__, url_prefix="/log_records")
+
+
+@blueprint.route("/add", methods=["POST"], strict_slashes=False)
+def add_log_record():
+    """
+    Add a log record
+    """
+    log_record = request.json
+    try:
+        created_log = log_records_service.add_record(log_record)
+        return jsonify(created_log), 201
+    except Exception as e:
+        error_message = getattr(e, "message", None)
+        return jsonify({"error": (error_message if error_message else str(e))}), 500
+
+
+@blueprint.route("/get", methods=["GET"], strict_slashes=False)
+def get_log_records():
+    """
+    Get all existing log records or input a start and end index of logs retrieved
+    """
```

Comment:
Little confused here - will we be able to get all existing log records? It looks like we're forced to provide a page number which means we'll get at most RESULTS_PER_PAGE entries back.